awesomerc.lua

```diff
@@ -62,7 +62,7 @@ myawesomemenu = {
    { "manual", terminal .. " -e man awesome" },
    { "edit config", editor_cmd .. " " .. awesome.conffile },
    { "restart", awesome.restart },
-   { "quit", function() awesome.quit() end },
+   { "quit", {{"I'm sure", function() awesome.quit() end}} },
```

Comment:
Oh it adds a submenu so if you intended to click restart and clicked quit nothing happens, like it usually happened to me, clicking the submenu triggers the quit. It's an alternative form of a confirmation dialog so I understand why you'd think it looks hacky.

Reply:
can you split the PR and only have the typo part? We usually avoid making any changes to the behavior of the default config. That's pretty minor, but yeah, that's a bit hacky. Users who want this can probably do it themselves. That's more something that would fit on stackoverflow (answering your own question) than in the default config.

Reply:
Done. It's now only the typo fixing commit.

Reply:
> That's more something that would fit on stackoverflow (answering your own question) than in the default config.

If we agree that we don't want this in the default awesomerc.lua, I think we should recommend @madprops to add to the [recipes](https://awesomewm.org/recipes/) as an alternative :smile: